<p>
    A <code>Scenario Outline</code> with an <code>Examples</code> section containing no data table will end up with the
    execution of zero scenario; that is unlikely the expected behavior. Thus, each <code>Examples</code> should contain
    a data table.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
Scenario Outline: Add product to cart
  Given I am a customer
  When I add &lt;number&gt; products to my cart
  Then I should see &lt;number&gt; products in my cart

  Examples:
</pre>

<h2>Compliant Solution</h2>
<pre>
Scenario Outline: Add product to cart
  Given I am a customer
  When I add &lt;number&gt; products to my cart
  Then I should see &lt;number&gt; products in my cart

  Examples:
    | number |
    | 1      |
    | 2      |
</pre>

<p>
    A <code>Scenario Outline</code> with an <code>Examples</code> section containing a data table with one single row
    will end up with the execution of zero scenario; that is unlikely the expected behavior. Thus, each
    <code>Examples</code> data table should contain at least two data rows.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
Scenario Outline: Add product to cart
  Given I am a customer
  When I add &lt;number&gt; products to my cart
  Then I should see &lt;number&gt; products in my cart

  Examples:
    | number |
</pre>

<h2>Compliant Solution</h2>
<pre>
Scenario Outline: Add product to cart
  Given I am a customer
  When I add &lt;number&gt; products to my cart
  Then I should see &lt;number&gt; products in my cart

  Examples:
    | number |
    | 1      |
    | 2      |
</pre>

<p>
    A <code>Scenario Outline</code> with an <code>Examples</code> section containing a data table with two rows will end
    up with the execution of a single scenario. Thus, either some data are missing or the <code>Scenario Outline</code>
    can be safely converted to a standard <code>Scenario</code>.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
Scenario Outline: Add product to cart
  Given I am a customer
  When I add &lt;number&gt; products to my cart
  Then I should see &lt;number&gt; products in my cart

  Examples:
    | number |
    | 1      |
</pre>

<h2>Compliant Solution</h2>
<pre>
Scenario Outline: Add product to cart
  Given I am a customer
  When I add &lt;number&gt; products to my cart
  Then I should see &lt;number&gt; products in my cart

  Examples:
    | number |
    | 1      |
    | 2      |
    
OR   

Scenario: Add product to cart
  Given I am a customer
  When I add 1 product to my cart
  Then I should see 1 product in my cart
</pre>
