<p>For readability reasons, code should be properly indented.</p>

<h2>Noncompliant Code Example</h2>
With default value: 2
<pre>
Feature: my feature...
  Scenario: my scenario...
    Given blabla...
      When blabla...  # Noncompliant
    Then blabla...
</pre>

<h2>Compliant Solution</h2>
With default value: 2
<pre>
Feature: my feature...
  Scenario: my scenario...
    Given blabla...
    When blabla...
    Then blabla...
</pre>
