<p>
    For a scenario to remain readable and understandable, the number of its steps should remain limited. The default
    threshold is 12 and it includes steps in <code>Background</code>.
</p>

<h2>Noncompliant Code Example</h2>
With a custom threshold of 8:
<pre>
Background:
  Given ...
  And ...
  And ...

Scenario: ...
  Given ...
  When ...
  Then ...
  And ...
  And ...
  And ...
</pre>
