<p>A scenario with a name:</p>
<ul>
    <li>Provides added value in terms of documentation and understanding</li>
    <li>Is easier to find out while debugging</li>
</ul>

<h2>Noncompliant Code Example</h2>
<pre>
Scenario:
  Given ...
  When ...
  Then ...
</pre>

<h2>Compliant Solution</h2>
<pre>
Scenario: Cannot register with a weak password
  Given ...
  When ...
  Then ...
</pre>

<h2>See</h2>
<ul>
    <li><a href="https://cucumber.io/blog/2016/07/01/cucumber-antipatterns-part-one" target="_blank">Cucumber
        anti-patterns (part one)</a></li>
</ul>
