<p>A feature with a name:</p>
<ul>
    <li>Provides added value in terms of documentation and understanding</li>
    <li>Is easier to find out while debugging</li>
</ul>

<h2>Noncompliant Code Example</h2>
<pre>
Feature:
  Scenario: ...
</pre>

<h2>Compliant Solution</h2>
<pre>
Feature: Displaying the appropriate welcome offer
  Scenario: ...
</pre>
