<p>A scenario with a name is:</p>
<ul>
    <li>more quickly and easily understandable</li>
    <li>easier to find out while debugging</li>
</ul>

<h2>Noncompliant Code Example</h2>
<pre>
Scenario:
  Given ...
  When ...
  Then ...
</pre>

<h2>Compliant Solution</h2>
<pre>
Scenario: Cannot register with a weak password
  Given ...
  When ...
  Then ...
</pre>

<h2>See</h2>
<ul>
    <li><a href="https://cucumber.io/blog/2016/07/01/cucumber-antipatterns-part-one">Cucumber anti-patterns (part
        one)</a></li>
</ul>
