<p>
    A file that does not define any feature is useless. It can either be safely removed or a feature should be
    added to this file.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
# Empty file
</pre>

<h2>Compliant Solution</h2>
<pre>
Feature: My feature...
  Scenario: My scenario #1
    ...
</pre>
