<p>
    The <code>Background</code> is supposed to set the context that is common to all the scenarios of the feature. Thus,
    it should only contain <code>Given</code> steps.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
Background:
  Given I am a customer
  And I am on the catalog page
  When I add a product to my cart    # Noncompliant
</pre>

<h2>Compliant Solution</h2>
<pre>
Background:
  Given I am a customer
  And I am on the catalog page
</pre>
