<p>
    In order to improve understandability, each step prefix type should be unambiguous.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
Scenario: Add product to cart
  And I am a customer                        # Noncompliant
  When I add a product to my cart
  Then I should see the product in my cart
  And I should see the update total amount
</pre>

<h2>Compliant Solution</h2>
<pre>
Scenario: Add product to cart
  Given I am a customer
  When I add a product to my cart
  Then I should see the product in my cart
  And I should see the update total amount
</pre>
