<p>
    If a tag is set to all scenarios, it should be set to the feature instead.
</p>

<h2>Noncompliant Code Example</h2>
<pre>
Feature: my feature

  @mytag
  Scenario: my scenario #1
    ...

  @mytag
  Scenario Outline: my scenario #2
    ...
</pre>

<h2>Compliant Solution</h2>
<pre>
@mytag
Feature: my feature

  Scenario: my scenario #1
    ...

  Scenario Outline: my scenario #2
    ...
</pre>
